wiki/Guides/BSS_Issues/tr.md

```diff
@@ -0,0 +1,38 @@
+---
+tags:
+- I can't submit my own beatmap!
+- beatmap submission system issues
+---
+
+# BSS sorunları nasıl çözümlenir
+
+Eğer ki **[Beatmap Submission System](/wiki/Glossary#bss)** (***BSS***) kendi beatmapinizi karşıya yüklemenize bu beatmape sahip olmadığınız için izin vermiyorsa aşağıdaki adımları uygulayın:
```

Comment:
```suggestion
Eğer ki **[Beatmap Gönderme Sistemi](/wiki/Glossary#bss)** (*Beatmap Submission System*, ***BSS***) kendi beatmapinizi karşıya yüklemenize bu beatmape sahip olmadığınız için izin vermiyorsa aşağıdaki adımları uygulayın:
```
this was translated in Rules article, so no reason to leave original version here